app/src/androidTest/java/org/mozilla/focus/activity/ImageSelectTest.java

```diff
@@ -134,6 +134,7 @@ public void ImageMenuTest() throws InterruptedException, UiObjectNotFoundExcepti
         TestHelper.hint.waitForExists(waitingTime);
         TestHelper.pressEnterKey();
         TestHelper.webView.waitForExists(waitingTime);
+        titleMsg.waitForExists(waitingTime);
```

Comment:
We should wrapp all waitForExists() calls int assertTrue(). Previously we were waiting for a bunch of things that never appeared which was just wasting time. :)